modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -1,6 +1,6 @@
-= Commands for the comments plugin
-:navtitle: Commands
-:description: Information on the commands provided with the comments plugin.
+= Commands and APIs for the comments plugin
```

Comment:
This needs updating again to include `Events`. It also needs updating in the `nav.adoc` file here: https://github.com/tinymce/tinymce-docs/blame/staging/docs-6/modules/ROOT/nav.adoc#L289